modules/zivid/__init__.py

```diff
@@ -22,3 +21,4 @@
 from zivid.settings_2d import Settings2D
 from zivid.camera_info import CameraInfo
 from zivid.camera_intrinsics import CameraIntrinsics
+from _zivid import Matrix4x4
```

Comment:
I don't know how the wrapper is designed, but to me it looks strange that only Matrix4x4 has underscore `_zivid` prefix.

Reply:
Ok I see. I think if @eskaur agrees with current approach then it is super. :+1:

Reply:
It is fine as long as the customer never has to write **`_zivid`**. That is still the case, right?

Reply:
Actually, @apartridge has a point here. For everything else we have a front-end class with actual Python source-code under `modules/zivid`. This is useful because the source code in that directory also serves as documentation. You class `_zivid.Matrix4x4` does not provide a docstring, for example.

So I think you should make a file `modules/zivid/matrix.py` that implements a class `zivid.Matrix4x4` that is a thin wrapper around `_zivid.Matrix4x4` with docstrings on all methods.

Reply:
@eskaur How about I add docstrings in the pybind code as well? It seems like an overkill to create a class just for the docstrings.

Reply:
The philosophy is that the user should not even notice that we are using pybind11. It is an implementation detail.
If they open a sample that uses zivid-python, they should be able to right-click->"Go to source" and end up in readable Python code. If we only expose the pybind11 stuff directly, there is no source code with docstrings to show.

Reply:
@eskaur Understandable. I've added a python class to adapt this one, including the docstrings, in `zivid.matrix4x4`.

Reply:
Nice!